pkg/virt-handler/device-manager/mediated_device.go

```diff
@@ -351,7 +193,7 @@ func discoverPermittedHostMediatedDevices(supportedMdevsMap map[string]string) m
 	return mdevsMap
 }
 
-func (dpi *MediatedDevicePlugin) healthCheck() error {
+func (dpi *MediatedDevicePlugin) HealthCheck() error {
```

Comment:
Why this function need to be exported?

Reply:
it didn't need to indeed, must've missed it when this method was still in the interface